src/Provider/Photon/Photon.php

```diff
@@ -158,6 +167,25 @@ public function getName(): string
         return 'photon';
     }
 
+    /**
+     * @param string|array<int, string>|null $filters
+     */
+    private function buildOsmTagFilterQuery($filters): string
```

Comment:
```suggestion
    private function buildOsmTagFilterQuery(?string $filters): string
```

See comment in `PhotonTest.php`.